File: adjacentElementsProduct/adjacentElementsProduct.js
function adjacentElementsProduct(arr) {
    let max = 0
    for (let i = 0; i < arr.length - 1; i++) {
        temp = arr[i] * arr[i + 1];
        if (temp > max) {
            max = temp;
        }
    }
    return max;
}
console.log(adjacentElementsProduct([3, 6, -2, -5, 7, 3]));

File: adjacentElementsProduct/adjacentElementsProduct.py
def adjacentElementsProduct(arr):
    num = 0
    for i in range(0, len(arr)-1):
        temp = arr[i] * arr[i+1]
        if(temp > num):
            num = temp
    return num


print(adjacentElementsProduct([3, 6, -2, -5, 7, 3]))
